app/controllers/billDiscountSuppliers.js

```diff
@@ -88,16 +93,24 @@ module.exports = {
             }
           })
           .catch((e) => {
-            console.log(e);
+            console.log("Error In fetching seller profile is>>>>", e);
           });
 
-        let supplier = await BillDiscountSuppliers.create({
-          invited_by: profile.uuid,
-          status: "pending",
-          ...item,
-        });
+        let supplier = null;
+        const [inviteExists, inviteStatus] = await checkExistingInvites(
+          profile.uuid,
+          item.profile_uuid
+        );
+        if (inviteStatus === "pending") {
+          pendingCount = pendingCount + 1;
+        }
+        if (!inviteExists) {
+          inviteCount = inviteCount + 1;
+          supplier = await upsertInvite(profile.uuid, item.profile_uuid, item);
+        } else {
```

Comment:
@ShreyDhyani Is this else block required here?

Reply:
Removed